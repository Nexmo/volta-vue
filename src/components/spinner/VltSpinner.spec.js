import { mount } from '@vue/test-utils';
import { VltSpinner } from '..';

describe('vlt-spinner', () => {
    
    test('renders correctly with no props', () => {
        const wrapper = mount(VltSpinner)
        expect(wrapper.element).toMatchSnapshot();
    });

    test('renders correctly with if small', () => {
        const wrapper = mount(VltSpinner, {
            propsData: {
                small: true
            }
        })
        expect(wrapper.element).toMatchSnapshot();
    });

    test('renders correctly if smaller', () => {
        const wrapper = mount(VltSpinner, {
            propsData: {
                smaller: true
            }
        })
        expect(wrapper.element).toMatchSnapshot();
    });

    test('renders correctly if white', () => {
        const wrapper = mount(VltSpinner, {
            propsData: {
                white: true
            }
        })
        expect(wrapper.element).toMatchSnapshot();
    });
})
