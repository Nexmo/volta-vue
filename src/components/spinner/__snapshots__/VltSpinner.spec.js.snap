// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vlt-spinner renders correctly if smaller 1`] = `
<div
  class="Vlt-spinner Vlt-spinner--smaller"
/>
`;

exports[`vlt-spinner renders correctly if white 1`] = `
<div
  class="Vlt-spinner Vlt-spinner--white"
/>
`;

exports[`vlt-spinner renders correctly with if small 1`] = `
<div
  class="Vlt-spinner Vlt-spinner--small"
/>
`;

exports[`vlt-spinner renders correctly with no props 1`] = `
<div
  class="Vlt-spinner"
/>
`;
